tools/java-iceberg-cli/src/main/java/iceberg_cli/IcebergConnector.java

```diff
@@ -194,6 +195,18 @@ public boolean createTable(Schema schema, PartitionSpec spec, boolean overwrite)
      * {
      *  "drop":["c1","c2"]
      * }
+     * 
+     * The property changes are expected in the following format:
+     * SETTING key/value property
+     * {
+     *  "set_prop":[
+     *      {"key":"p1","value":"v1"}
+     *   ]
+     * }
+     * REMOVING property key
+     * {
+     *  "rm_prop":["p1","p2"]
+     * }
```

Comment:
I suggest spelling out the operations names:
```suggestion
     * The property changes are expected in the following format:
     * SETTING key/value property
     * {
     *  "set_property":[
     *      {"key":"p1","value":"v1"}
     *   ]
     * }
     * REMOVING property key
     * {
     *  "remove_property":["p1","p2"]
     * }
```